stress/common/src/main/java/alluxio/stress/common/GeneralBenchSummary.java

```diff
@@ -38,16 +36,7 @@ public void setThroughput(float throughput) {
   }
 
   /**
-   * @return the errors
+   * @return the error information
    */
-  public Map<String, List<String>> getErrors() {
-    return mErrors;
-  }
-
-  /**
-   * @param errors the errors
-   */
-  public void setErrors(Map<String, List<String>> errors) {
-    mErrors = errors;
-  }
+  public abstract List<String> collectErrors();
```

Comment:
`getErrorsFromAllNodes()`

Reply:
I named it to collectErrorsFromAllNodes().
It seems that for JSONserializer, getXXX will have special meaning, and it should be in pair with a parameter. In order to use the JSON and use graph generator, I rename the method.